linode/instancenetworking/framework_datasource_schema.go

```diff
@@ -0,0 +1,68 @@
+package instancenetworking
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var networkObjectType = types.ObjectType{
+	AttrTypes: map[string]attr.Type{
+		"address":     types.StringType,
+		"gateway":     types.StringType,
+		"prefix":      types.Int64Type,
+		"rdns":        types.StringType,
+		"region":      types.StringType,
+		"subnet_mask": types.StringType,
+		"type":        types.StringType,
+	},
+}
```

Comment:
In the latest API, the response include an extra attribute `public`. Although we have the key values to show whether it's public (e.g., `ipv4.public` only contains public IP and `ipv4.private` only contains private IP), can we also include it here for user's convenient?

Reply:
I don't see an issue with keeping the datasource schema aligned with the current state of the API. I also noticed that I was missing `LinodeID` so I added that as well. Does anyone else have any thoughts?